src/SDKs/CognitiveServices/Management.CognitiveServices/Microsoft.Azure.Management.CognitiveServices.nuspec

```diff
@@ -0,0 +1,62 @@
+        <dependency id="Microsoft.Rest.ClientRuntime.Azure" version="[3.3.5, 4.0.0)" />
+        <dependency id="Microsoft.Rest.ClientRuntime" version="[2.3.5, 3.0.0)" />
+      </group>
+      <group targetFramework=".NETStandard1.4">
+        <dependency id="Microsoft.NETCore.Platforms" version="[1.1.0, )" />
+        <dependency id="NETStandard.Library" version="[1.6.1, )" />
+        <dependency id="System.Collections" version="[4.3.0, )" />
+        <dependency id="System.Collections.Concurrent" version="[4.3.0, )" />
+        <dependency id="System.Diagnostics.Debug" version="[4.3.0, )" />
+        <dependency id="System.Diagnostics.Tools" version="[4.3.0, )" />
+        <dependency id="System.Diagnostics.Tracing" version="[4.3.0, )" />
+        <dependency id="System.Globalization" version="[4.3.0, )" />
+        <dependency id="System.IO" version="[4.3.0, )" />
+        <dependency id="System.IO.Compression" version="[4.3.0, )" />
+        <dependency id="System.Linq" version="[4.3.0, )" />
+        <dependency id="System.Net.Http" version="[4.3.0, )" />
+        <dependency id="System.Net.Primitives" version="[4.3.0, )" />
+        <dependency id="System.ObjectModel" version="[4.3.0, )" />
+        <dependency id="System.Reflection" version="[4.3.0, )" />
+        <dependency id="System.Reflection.Extensions" version="[4.3.0, )" />
+        <dependency id="System.Reflection.Primitives" version="[4.3.0, )" />
+        <dependency id="System.Resources.ResourceManager" version="[4.3.0, )" />
+        <dependency id="System.Runtime" version="[4.3.0, )" />
+        <dependency id="System.Runtime.Extensions" version="[4.3.0, )" />
+        <dependency id="System.Runtime.InteropServices" version="[4.3.0, )" />
+        <dependency id="System.Runtime.InteropServices.RuntimeInformation" version="[4.3.0, )" />
+        <dependency id="System.Runtime.Numerics" version="[4.3.0, )" />
+        <dependency id="System.Text.Encoding" version="[4.3.0, )" />
+        <dependency id="System.Text.Encoding.Extensions" version="[4.3.0, )" />
+        <dependency id="System.Text.RegularExpressions" version="[4.3.0, )" />
+        <dependency id="System.Threading" version="[4.3.0, )" />
+        <dependency id="System.Threading.Tasks" version="[4.3.0, )" />
+        <dependency id="System.Xml.ReaderWriter" version="[4.3.0, )" />
+        <dependency id="System.Xml.XDocument" version="[4.3.0, )" />
+        <dependency id="System.Runtime.Serialization.Primitives" version="[4.1.1, )" />
+        <dependency id="Microsoft.Rest.ClientRuntime.Azure" version="[3.3.5, 4.0.0)" />
+        <dependency id="Microsoft.Rest.ClientRuntime" version="[2.3.5, 3.0.0)" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="bin\release\net452\Microsoft.Azure.Management.CognitiveServices.dll" target="lib\net452" />
+    <file src="bin\release\netstandard1.4\Microsoft.Azure.Management.CognitiveServices.dll" target="lib\netstandard1.4" />
```

Comment:
@felixwa a good way to make this change is good is to create a signed nuget package using the signing job to create a signed package and test out in SDK and PowerShell scenarios.